templates/jupyterhub_config.py.epp

```diff
@@ -14,19 +14,25 @@ c.JupyterHub.hub_ip = '<%= $jupyterhub::hub_ip %>'
 <% } -%>
 
 <% if $jupyterhub::admin_access { -%>
-c.JupyterHub.admin_access = <%= $jupyterhub::admin_access %>
+c.JupyterHub.admin_access = True
+<% } else { -%>
+c.JupyterHub.admin_access = False
 <% } -%>
 
 <% if $jupyterhub::authenticator_admin_users { -%>
-c.Authenticator.admin_users = set(<%= $jupyterhub::authenticator_admin_users %>)
+c.Authenticator.admin_users = set([<% $jupyterhub::authenticator_admin_users.each |$admin_user| { -%> <%= "'$admin_user', "-%> <% } -%>])
```

Comment:
This can be fixed with a join and a separator I guess:
```puppet
join($jupyterhub::authenticator_admin_users,",")
```

Reply:
I tried this, but because the jupyterhub config is a python script, without the strings being bound in quotes, the config will fail to import as it assumes the usernames are variables rather than strings. 

Puppet seems to pass them along without quoting them and couldn't find a good way of doing that. But let's discuss if there's a good way of doing that!

Reply:
The fact that it's a python script is also why I do the implicit True/False checks, not only does it cover any mistakes in boolean logic from Puppet, but puppets bools won't work in Python anyway because of capitalisation.

Reply:
I fixed it with a join (playing with quotes...)
```puppet
c.Authenticator.admin_users = set(['<%= $jupyterhub::authenticator_admin_users.join("','") %>'])
```

Reply:
Ok nice, I'll amend the pull request.